saleor/static/dashboard-next/js/misc.js

```diff
@@ -0,0 +1,11 @@
+export const screenSizes = {
+  sm: 600,
+  md: 992,
+  lg: 1200
+};
+
+export function createQueryString(currentQs, newData) {
```

Comment:
Is mixing constants and utility functions in one file a good idea?

Reply:
I'll get rid of both eventually, creating two temporary files did not seem like a better idea.